cohere-cookbooks/Semantic search with Cohere embeddings/Wikipedia_Semantic_Search_With_Cohere_Embeddings_Archives.ipynb

```diff
@@ -0,0 +1,554 @@
+      "\n",
+      "After the war, Turing continued his work in computing and artificial intelligence. He designed the Automatic Computing Engine (ACE), one of the first designs for a stored- \n",
+      "\n",
+      "\n",
+      "Result of API call 2 :\n",
+      "Alan Turing was a pioneering figure in the field of computing and cryptography. He is widely regarded as one of the most influential thinkers of the 20th century for his groundbreaking work in mathematics, computer science, and cryptography during World War II. \n",
+      "\n",
+      "As mentioned in the provided information, Turing created the theoretical concept of a \"Turing machine\" in 1936, which was a significant contribution to the emerging field of computing. This imaginary machine introduced the idea of a computer program and set the foundation for modern computing. \n",
+      "\n",
+      "During World War II, Turing played a crucial role in breaking German military codes, most notably those generated by the Enigma machine. His work at Bletchley Park, Britain's code-breaking center, was instrumental in providing Allied forces with crucial intelligence that greatly aided their efforts during the war. \n",
+      "\n",
+      "Despite his remarkable contributions, Turing's life took a tragic turn due to the societal attitudes and legal framework of his time. He was convicted of \"gross inde \n",
+      "\n",
+      "\n",
+      "Result of API call 3 :\n",
+      "Alan Turing was a pioneering British mathematician, computer scientist, and cryptanalyst. He is widely regarded as one of the most influential figures in the development of theoretical computer science and is often referred to as the \"father of computer science and artificial intelligence.\"\n",
+      "\n",
+      "As you mentioned, Turing is known for his concept of the Turing machine, which he introduced in 1936. The Turing machine was a theoretical device that served as a simple model for a general-purpose computer. It could perform any computation if given the appropriate instructions, providing a framework for the design of modern computers.\n",
+      "\n",
+      "During World War II, Turing played a crucial role in breaking German military codes, including those generated by the Enigma machine. His work at Bletchley Park, the central site for British code-breaking operations, significantly contributed to the Allied victory.\n",
+      "\n",
+      "After the war, Turing continued his work in computer science and mathematics. He designed the Automatic Computing Engine (ACE), one of the first designs for a \n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Send the same query to the Command R+ model using the Bedrock converse API but add the results of the semantic search as context\n",
+    "conversation = [\n",
```

Comment:
I believe Command R models do not support conversation turns... please check my knowledge but if that's the case we may want to note that for the user.

Reply:
this is correct does not support multi conversation turns on Bedrock

Reply:
Got it! I added a note clarifying that the user needs to use the Command R model (not the Command R+ model) if they want to include a chat_history/conversation with multiple turns.